relay-server/src/services/store.rs

```diff
@@ -347,6 +349,10 @@ impl StoreService {
             }
         }
 
+        if has_feedback && !attachments.is_empty() {
+            metric!(counter(RelayCounters::FeedbackAttachments) += attachments.len() as i64);
```

Comment:
What exactly do we want to count here, the number of feedback envelopes that have an attachment, or the total number of attachments? Can a single feedback envelope contain multiple screenshots?

Reply:
Thanks for the suggestions! I'm thinking it might be better to add a metadata field to the payload instead. Will close and come back to this if not

Reply:
> What exactly do we want to count here, the number of feedback envelopes that have an attachment, or the total number of attachments? Can a single feedback envelope contain multiple screenshots?

We'd just be counting the total number of attachments for now. Currently we only support 1 ss/attachment per feedback, don't expect this to change for a while